src/app/content/search/hooks/index.ts

```diff
@@ -29,20 +31,31 @@ export const requestSearchHook: ActionHookBody<typeof requestSearch> = (services
     searchStrategy: 's1',
   });
 
+  const params = queryString.parse(services.history.location.search);
+  params.query = payload;
+  services.history.replace({
+    hash: services.history.location.hash,
+    search: queryString.stringify(params),
+  });
```

Comment:
why do we need this block up here? won't all the url stuff be handled by the navigation after receiveSearchResults?

Reply:
looks likes that were some leftovers after my testing